bundles/org.openhab.binding.mideaac/src/main/java/org/openhab/binding/mideaac/internal/handler/MideaACHandler.java

```diff
@@ -0,0 +1,399 @@
+            } catch (MideaConnectionException | MideaException e) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+            }
+            return;
+        }
+        try {
+            Response lastresponse = connectionManager.getLastResponse();
+            if (channelUID.getId().equals(CHANNEL_POWER)) {
+                connectionManager.sendCommand(CommandHelper.handlePower(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_OPERATIONAL_MODE)) {
+                connectionManager.sendCommand(CommandHelper.handleOperationalMode(command, lastresponse),
+                        callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_TARGET_TEMPERATURE)) {
+                connectionManager.sendCommand(CommandHelper.handleTargetTemperature(command, lastresponse),
+                        callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_FAN_SPEED)) {
+                connectionManager.sendCommand(CommandHelper.handleFanSpeed(command, lastresponse, config.version),
+                        callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_ECO_MODE)) {
+                connectionManager.sendCommand(CommandHelper.handleEcoMode(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_TURBO_MODE)) {
+                connectionManager.sendCommand(CommandHelper.handleTurboMode(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_SWING_MODE)) {
+                connectionManager.sendCommand(CommandHelper.handleSwingMode(command, lastresponse, config.version),
+                        callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_SCREEN_DISPLAY)) {
+                connectionManager.sendCommand(CommandHelper.handleScreenDisplay(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_TEMPERATURE_UNIT)) {
+                connectionManager.sendCommand(CommandHelper.handleTempUnit(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_SLEEP_FUNCTION)) {
+                connectionManager.sendCommand(CommandHelper.handleSleepFunction(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_ON_TIMER)) {
+                connectionManager.sendCommand(CommandHelper.handleOnTimer(command, lastresponse), callbackLambda);
+            } else if (channelUID.getId().equals(CHANNEL_OFF_TIMER)) {
+                connectionManager.sendCommand(CommandHelper.handleOffTimer(command, lastresponse), callbackLambda);
+            }
+        } catch (MideaConnectionException | MideaAuthenticationException e) {
+            logger.warn("Unable to proces command: {}", e.getMessage());
```

Comment:
When an authentication exception occurs, i guess it is permanenet. When a timeout occurs, it is transient and the binding should recover. You might want to handle both cases seperate.
```suggestion
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
```

Reply:
Ok This was part of the code you rewrote and have never seen an exception, so don't know what else to say.